tests/imodels-clients-tests/src/integration/management/ChangesetOperations.test.ts

```diff
@@ -28,6 +29,16 @@ describe("[Management] ChangesetOperations", () => {
 
     const reusableTestIModelProvider = container.get(ReusableTestIModelProvider);
     testIModel = await reusableTestIModelProvider.getOrCreate();
+
+    testIModel.namedVersions.some((version) => {
```

Comment:
I tried a few ways of testing changesets and the best one seemed to be creating a named version with group id. Open to any suggestions.. 

Also, not sure if this code should go here or in reusable iModel provider.

Reply:
it's sufficient, but I think writing 2 separate tests to check if a changeset has named version and group id is a bit redundant.. currently reusable iModel metadata does not contain information about changesets, therefore to know whether it's in a group or not, is done by filtering, which is not the best..

Maybe we should add `changesets` property to `ReusableIModelMetadata`?